src/Bundle/CoreBundle/Form/StateType.php

```diff
@@ -0,0 +1,293 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: stefan.kamsker
+ * Date: 23.09.18
+ * Time: 20:02
+ */
+
+namespace UniteCMS\CoreBundle\Form;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Validator\Constraints\Callback;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
+use UniteCMS\CoreBundle\Model\State;
+
+class StateType extends AbstractType implements DataTransformerInterface
+{
+
+    private $settings;
+
+    /**
+     * @var TranslatorInterface $translator
+     */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $this->setSettings($options['settings']);
+
+        $builder->add('state', HiddenType::class,
+            [
+                'label' => '',

```

Comment:
Label should be false if you want to hide it (https://symfony.com/doc/current/reference/forms/types/text.html#label)

Reply:
I think the hidden field type does not have a label option: https://symfony.com/doc/current/reference/forms/types/hidden.html